### PR TITLE
updating to integration tests and solution paths

### PR DESCRIPTION
**Pull request by dguerin**
* Update Test Paths
* Project rename from template for integration tests (template rename wasn't working)
* Changed .sln project ordering for better startup experience (VS should pick WebAPIService as default project so you can just F5 it and get into the web project debug on first project open).
* Fixing up target platform

### CODE REVIEW

File: src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+using Eshopworld.Tests.Core;
+using Xunit;
+
+namespace WebAPIService.Integration.Tests
+{
+    public class IntegrationTest1
+    {
+        [Fact, IsIntegration]
```
Comment (VaclavK) [inline src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: @djfr do we want people to start diverting to CI/CD (levelX) attributes?

File: src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        public void Test1()
```
Comment (VaclavK) [inline src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: empty constructor?

File: src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs (hunk 1 above)
Comment (djfr) [reply src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: nope!

File: src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs (hunk 2 above)
Comment (djfr) [reply src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: it's not a ctor, it's the test itself 😄

Comment (dguerin) [reply src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: yeah, I can add more in if it's generic enough but won't add much more for this version.

File: src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+using Eshopworld.Tests.Core;
+using Xunit;
+
+namespace WebAPIService.Integration.Tests
+{
```
Comment (djfr) [inline src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: Remove all namespaces from xUnit tests and add the Resharper disable warning comment for the "No namespace" warning.

**Comment by djfr** [comment]
Since we're re-visiting this, it would be nice to add an example of an integration test that reads configuration, so they have initial guidance on how to use it.

File: src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs (hunk 2 above)
Comment (VaclavK) [reply src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: aah... the GitHub view separated the attributes from the method... ignore me (facepalm)

**Comment by dguerin** [comment]
> Since we're re-visiting this, it would be nice to add an example of an integration test that reads configuration, so they have initial guidance on how to use it.

This PR is more about fixing up a spelling mistake in the integration name so that the template transform would work. Wasn't planning on doing more work with it other than that. But I can backlog something and do it  in the coming sprints.

**Comment by dguerin** [comment]
@VaclavK @arturek I added in the fix for local 5 cluster deployments to just a single node as well. (And tested it :) )

File: src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs (hunk 3 above)
Comment (djfr) [reply src/content/src/tests/WebAPIService.Integration.Tests/IntegrationTest1.cs]: By default xUnit uses the namespace to compose the test name, and that creates names that are too long and the start of the name is useless. The meaningful information is the class structure part of the naming composition, that with the namespace sits in the middle/end parts of the test name.

**Pull request merged**